tests/tensorflow/test_tensorflow_model_export.py

```diff
@@ -30,8 +30,8 @@
 from tests.helper_functions import mock_s3_bucket  # pylint: disable=unused-import
 
 SavedModelInfo = collections.namedtuple(
-        "SavedModelInfo",
```

Comment:
BTW, super small nit but in general it's best to avoid these sort of no-op whitespace changes if possible (did your editor make them or something?). Not a big deal, it just makes the `git blame` harder to trace through